lib/rpc/handler/FallbackHandler.ts

```diff
@@ -0,0 +1,81 @@
+import Logger from 'bunyan'
+import { ChainId } from '@uniswap/sdk-core'
+import { getProviderId } from '../utils'
+import { ProviderHealthStateRepository } from '../ProviderHealthStateRepository'
+import { ProviderHealthStateDynamoDbRepository } from '../ProviderHealthStateDynamoDbRepository'
+import { ProviderHealthState } from '../ProviderHealthState'
+import { metric, MetricLoggerUnit } from '@uniswap/smart-order-router'
+
+interface AlarmEvent {
+  alarmName: string
+  state: string
+  previousState: string
+  providerId: string
+  reason: string
+}
+
+export class FallbackHandler {
+  private healthStateRepository: ProviderHealthStateRepository
+  private log: Logger
+
+  constructor(log: Logger) {
+    const dbTableName = process.env.PROVIDER_HEALTH_STATE_DB_TABLE_NAME
+    if (!dbTableName) {
+      throw new Error('Missing DB_TABLE_NAME in env var')
+    }
+    this.healthStateRepository = new ProviderHealthStateDynamoDbRepository(dbTableName, log)
+    this.log = log
+  }
+
+  get handler() {
```

Comment:
It would be great if we are explicit in the return type, as well as the lambda event param type. Since this handler is the handler that inherits the lambda handler, being more strongly typed benefits from that.